src/main/java/org/codehaus/mojo/versions/DisplayPluginUpdatesMojo.java

```diff
@@ -304,36 +305,21 @@ else if ( event.isEndElement() )
                 }
             }
         }
-        catch ( NullPointerException e )
-        {
-            // guess there are no plugins here
-        }
-        try
+        for ( Profile profile : model.getProfiles() )
```

Comment:
Same case here, you can still get NullPointers here, if `model.getProfiles()` returns null.